data/base-spells.yaml

```diff
@@ -451,6 +451,15 @@ barb_abilities:
     start_command: meditate staunch
     activated_message: ^You .* to meditate
     expired_message: heart quickens and blood rushes back
+  Unyielding:
```

Comment:
Unyielding requires an expired message so it can set the flag appropriately to reuse when worn off.

Reply:
It doesn’t have one that I can find. See initial comment.